ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-07.xml

```diff
@@ -72,56 +72,214 @@
+			CPU, memory and power.</t>
+
+			<t>The BDP extension proposes three new parameters that
+			help a connection startup in aminimal number of
+			RTTs:<list style="symbols">
+
+				<t>recon_bytes_in_flight (0x000X): The bytes in
+				flight measured on the previous connection by
+				the server. Integer numbder of bytes.
+				Using the bytes_in_flight defined in <xref
+				target="I-D.ietf-quic-recovery"></xref>,
+				recon_bytes_in_flight can be set to
+				bytes_in_flight.</t>
+
+				<t>recon_min_rtt (0x000X): The minimum RTT
+				measured on the previous connection by the
+				server. Integer number of milliseconds. Using
+				the min_rtt defined in <xref
+				target="I-D.ietf-quic-recovery"></xref>,
+				recon_min_rtt can be set to min_rtt.</t>
+
+				<t>recon_max_pkt_number (0x000X): The maximum
+				amount of packets that the server is allowed to
+				send when reconnecting. It is an integer in
+				unit of packets.</t>
+			</list></t>
 		</section>
 
 		<section anchor="sec:bdp_activation" title="Extension activation">
-			<t>BDP extension is protected by the mechanism which protect the exchange of the 0-RTT transport parameters. Currently, in section 7.3.1 of <xref target="I-D.ietf-quic-transport"></xref>, a client which activates 0-RTT sends back the transport parameters (initial_max_data ...) received from the server during the previous connection.</t>
-			<t>The client MUST read the parameters in the BDP metadata extension but can not change them.</t>
-			<t>Accept: A client which activates ingress optimization must send back the transport parameters of the BDP metadata extension that it received from the server during the previous connection.</t>
-			<!-- <t>Tune: A client may send back a value lower than the recon_max_pkt_number received from the server.</t> -->
-			<t>Refuse: A client which does support ingress optimisation drop the extension signal. A client which disagrees with the extension parameters received from the server refuses the optimization. A limited-resource client ignores the extension.</t>
+			<t>The BDP extension is protected by the same mechanism
+			that protects the exchange of the 0-RTT transport
+			parameters.  A client which activates 0-RTT data sends
+			back the transport parameters received from the server
+			during the previous connection (see Section 7.3.1 of
+			<xref target="I-D.ietf-quic-transport"></xref>).  </t>
+
+			<t>The client MUST read the parameters in the BDP
+			metadata extension but can not change them.</t>
+
+			<t>Accept: A client which activates ingress
+			optimization must send back the transport parameters of
+			the BDP metadata extension that it received from the
+			server during the previous connection.</t>
+
+			<t>Refuse: A client which does not support ingress
+			optimisation drops the extension signal. A client which
+			disagrees with the extension parameters received from
+			the server refuses the optimization.</t>
```

Comment:
I wonder if "disagrees" is really the correct word here? Perhaps: "the client MAY use the extension parameters...." and then say "A client could choose not to use these parameters."